theme.json

```diff
@@ -612,23 +612,9 @@
 			},
 			"core/pullquote": {
 				"border": {
-					"bottom": {
-						"style": "none",
-						"width": "0px"
-					},
-					"left": {
-						"style": "none",
-						"width": "0px"
-					},
 					"radius": "var(--wp--preset--spacing--20)",
-					"right": {
-						"style": "none",
-						"width": "0px"
-					},
-					"top": {
-						"style": "none",
-						"width": "0px"
-					}
+					"style": "solid",
```

Comment:
This does not need to be added, as the default design of the pullquote block does not have a border. Please remove these two lines.

Reply:
Hi @luminuu 

Done,

Please check it

Thanks,